env0/resource_environment.go

```diff
@@ -1113,6 +1115,21 @@ func resourceEnvironmentImport(ctx context.Context, d *schema.ResourceData, meta
 	d.Set("deployment_id", environment.LatestDeploymentLogId)
 	setEnvironmentSchema(d, environment, environmentConfigurationVariables)
 
+	if environment.IsRemoteBackend != nil {
+		d.Set("is_remote_backend", *environment.IsRemoteBackend)
+	}
+
+	if environment.AutoDeployOnPathChangesOnly != nil {
+		d.Set("auto_deploy_on_path_changes_only", *environment.AutoDeployOnPathChangesOnly)
+	}
+
+	d.Set("is_inactive", false) // default is false.
+	if environment.IsArchived != nil {
+		d.Set("is_inactive", *environment.IsArchived)
+	}
+
+	d.Set("force_destroy", false)
```

Comment:
force_destory is always false when importing.
It's a schema field only.

Reply:
why is that always false when importing?

Reply:
Missing `default: false` in the schema.
Again... historical mistakes... in general it's a good practice to always set a default for type bool